src/main/java/mods/eln/Eln.java

```diff
@@ -5284,9 +5572,9 @@ private void recipeLampSocket() {
             'I', new ItemStack(Items.iron_ingot));
 
         addRecipe(findItemStack("Lamp Socket B Projector", 3),
-            " I",
```

Comment:
No matter what I do, I can only craft Lamp Socket B. Wut.

Reply:
Or was it A? I can't recall. But I cannot craft one of them.

Reply:
Yeah this one was a real **NANI!** bug. It's fixed now, I just replaced the iron in the A recipe with iron cable.